stammer.py

```diff
@@ -146,34 +146,75 @@ def create_output_audio(best_matches, modulator_audio, carrier_frames, modulator
 
     wavfile.write(TEMP_DIR / 'out.wav', INTERNAL_SAMPLERATE, output_audio)
 
+COMMON_AUDIO_EXTS = [
```

Comment:
small nitpick, but usually prefer to have global constants all together near the top of the file

Reply:
I thought it would be cluttery against all the constants dedicated to hard processing. Oh well /shrug?